interop-tests/src/lib.rs

```diff
@@ -29,12 +29,16 @@ pub async fn run_test(
     let redis_client = RedisClient::new(redis_addr).context("Could not connect to redis")?;
 
     // Build the transport from the passed ENV var.
-    let (boxed_transport, local_addr) = build_transport(local_key, ip, transport)?;
+    let (boxed_transport, local_addr) = build_transport(local_key.clone(), ip, transport)?;
     let mut swarm = swarm_builder(
         boxed_transport,
         Behaviour {
             ping: ping::Behaviour::new(ping::Config::new().with_interval(Duration::from_secs(1))),
             keep_alive: keep_alive::Behaviour,
+            identify: identify::Behaviour::new(identify::Config::new(
+                "/interop-tests".to_owned(),
+                local_key.public(),
+            )),
```

Comment:
In case you want to merge with this change included, please add a comment.

Reply:
I don't think we have a choice unless we wait until `nim-libp2p` patches its implementation.

Reply:
What sort of comment would you like to see?

Reply:
A link to https://github.com/status-im/nim-libp2p/issues/924. Once that is fixed we can remove the above.